CoreFoundation/Base.subproj/CFPlatform.c

```diff
@@ -175,6 +176,11 @@ const char *_CFProcessPath(void) {
     }
     return __CFProcessPath;
 }
+
+Boolean _CFIsMainThread(void) {
+    return syscall(SYS_gettid) == getpid();
```

Comment:
where is this function used? we already have a shim definition of pthread_main_np